src/PostProcess/babylon.ssao2RenderingPipeline.ts

```diff
@@ -0,0 +1,369 @@
+﻿module BABYLON {
+    export class SSAO2RenderingPipeline extends PostProcessRenderPipeline {
+        // Members

```

Comment:
IsSupported will be added, but I'm not sure about the fallback, SSAO v2 and v1 have totally different shaders, and parameter uniforms are not the same. 

Maybe just offering the possibility for devs to test that the effect is supported would be enough, what do you think ?